core/src/main/java/com/microsoft/applicationinsights/internal/config/ApplicationInsightsXmlConfiguration.java

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
I thought about it but I think that it makes things clearer while you work with it later on in the code